corehq/apps/reports/static/reports/javascripts/reports.config.js

```diff
@@ -113,27 +112,11 @@ var HQReport = function (options) {
     };
 
     var checkFilterAccordionToggleState = function () {
-        var _setShowFilterCookie = function (show) {
-            var showStr = show ? 'in' : '';
-            $.cookie(self.toggleFiltersCookie, showStr, {path: self.urlRoot, expires: 1});
-        };
-        
-        if (!$.cookie(self.toggleFiltersCookie) && $.cookie(self.toggleFiltersCookie) !== '') {
-            // default to showing filters
-            // (When filters should be hidden, $.cookie(self.toggleFiltersCookie) === '')
-            _setShowFilterCookie(true);
-        }
-        $(self.filterAccordion).addClass($.cookie(self.toggleFiltersCookie));
-
-        if ($.cookie(self.toggleFiltersCookie) == 'in') {
-            $(self.toggleFiltersButton).button('close');
-        } else {
-            $(self.toggleFiltersButton).button('open');
-        }
+        $(self.filterAccordion).addClass('in');
+        $(self.toggleFiltersButton).button('close');
```

Comment:
It just means show the text state for that button named 'close'. http://getbootstrap.com/javascript/#buttons